spec/bolt/executor_spec.rb

```diff
@@ -214,6 +214,77 @@ def mock_node_results
     end
   end
 
+  context 'running a task with per-target params' do
+    let(:target_mapping) do
+      targets.each_with_object({}) { |target, map| map[target] = task_arguments }
```

Comment:
Since the expectation also iterates over each target, I think this can use a target attribute instead of a static string to be more like the actual use case for this function without any extra work.